src/components/generic/floor-plan-map.tsx

```diff
@@ -708,25 +976,49 @@ export default class FloorPlanMap extends React.Component<Props, State> {
     return deviceGroupBounds;
   }
 
+  /**
+   * Find device points
+   *
+   * @param tempDevices list of device
+   * @param selectedDeviceGroup selected device group
+   * @param devicePoints list of device points
+   * @param tempAntennas list of antennas
+   */
+  // tslint:disable-next-line: max-line-length
+  private findDevicePoints = (tempDevices: ExhibitionDevice[], selectedDeviceGroup: ExhibitionDeviceGroup, devicePoints: LatLngExpression[], tempAntennas?: RfidAntenna[]) => {
```

Comment:
what is the purpose of this method? findX methods usually find and return something but this does not return anything